packages/components/src/internal/components/samples/models.ts

```diff
@@ -161,10 +162,8 @@ export interface SampleGridButtonProps {
     afterSampleDelete?: (rowsToKeep: any[]) => void;
     createBtnParentKey?: string;
     createBtnParentType?: string;
-    excludedCreateMenuKeys?: List<string>;
-    excludedManageMenuKeys?: SamplesManageButtonSections[];
-    hideButtons?: SamplesManageButtonSections[];
-    excludeStartJob?: boolean;
+    excludedManageMenuKeys?: SamplesEditButtonSections[];
+    hideButtons?: SamplesEditButtonSections[];
```

Comment:
```suggestion
    hiddenButtons?: SamplesEditButtonSections[];
```
I keep expecting `hideButtons` to be a boolean.

Reply:
Done, changed to hiddenButtons